admin_manual/configuration/database/db_conversion.rst

```diff
@@ -8,6 +8,44 @@ If you have an existing ownCloud installation which uses SQLite, and you want to
 
 .. note:: ownCloud Enterprise edition does not support SQLite.
 
+
+Preparation
+-----------
+
+Add to your config.php
+
+::
+
+	'mysql.utf8mb4' => true,
+
+Add / adjust in your /etc/mysql/mariadb.conf.d/50-server.cnf 
```

Comment:
``Add / adjust`` --> ``Add or adjust``
Add: You can do the same for ``mysql`` by replacing ``mariadb.conf.d`` with ``mysql.conf.d``